### DISCUSSION TITLE
Is there a way to find what process owns a stuck tooltip with System Informer?

### DISCUSSION THREAD

**Discussion by Almighty188** [winsiderss/systeminformer#2180] [Questions / Answers] [answered]
Is there a way to find what process owns a stuck tooltip with System Informer?  

![image](https://github.com/user-attachments/assets/54daa422-54a9-453b-8f69-a64f93ab1eb2)

**Comment by dmex** [comment, marked as answer]
Click and drag the Find Window button to the tooltip and it'll select the process that created the tooltip window.

**Reply by Almighty188** [reply]
Thanks @dmex!  Does SI have the function to send a WM_CLOSE to the window in question?

**Reply by enervated** [reply]
> Thanks @dmex! Does SI have the function to send a WM_CLOSE to the window in question?

Yes.

**Reply by Almighty188** [reply]
@enervated - Can you share how to send the WM_CLOSE to the window in question?

**Reply by enervated** [reply]
Sure. Start with dmex's instructions above:

1. Click and drag the "Find Window" button to the tooltip. It will select the process that created the tooltip window.
2. Hit Enter to open the Properties dialog for that process.
3. Go to the Windows tab on the Properties dialog.
4. Find the desired window (it may have the class name "tooltips_class32"), right click it, and select Close. (In some cases you may want to choose Destroy instead.)

**Reply by Almighty188** [reply]
@enervated - I think 4 is probably the hardest on figuring out which is the correct window in question if there are multiple class name entries and none have "tooltips_class32".  Many thanks again!

**Reply by diversenok** [reply]
You can click on `Highlight` or toggle `Visible` on/off to find the right window after you narrowed it down to a few

**Reply by Almighty188** [reply]
@diversenok - Thanks.  What does Highlight or toggling visible actually do as since I didn't have a current problem, I opened some random app like CPU-Z's .exe and I didn't notice the Highlight or Visible actually doing anything .  I guess for things like this, visuals would help.  Instead for Step 4, wouldn't choosing Find Window under the Windows tab and then drag it to the window/tooltip also work?

**Reply by enervated** [reply]
If "Visible" has a checkmark next to it, deselecting "Visible" should hide the window/element. Selecting "Highlight" should cause the element to flash a black border for a moment. In my experience, "Highlight" does not work with all elements.

If these tooltips are a frequent hassle, you could use NirCmd's "win hide" function to hide them semi-automatically or even automatically (with a scheduled task). It's hard to run Windows and not resort to a litany of kludges like that over time.

**Reply by Almighty188** [reply]
@enervated - The tooltip is like once in a blue moon so I just open up some program to test but I couldn't get either Visible and Highlight to do anything.  Is there a way to manually force a tooltip to try the Visible and Highlight?  Thanks for the pointer on Nircmd.  There is a tool called WinCheat which does the WM_CLOSE as mentioned at https://superuser.com/a/1545136
I was just wondering just how to do the same thing in System Informer.
